### ISSUE TITLE
Convert config.js to config.ts

### ISSUE THREAD

**Issue by ijemmao** [nkowaokwu/igbo_api#649]
## Background
Convert this file to TS
https://github.com/nkowaokwu/igbo_api/blob/f8718bae6e84206264bb3d3aa8ac5d7b6c981739/src/config.js#L1

**Comment by jibranabsarulislam** [comment]
I'd be interested in taking up this task after #653 and getting the project fully setup locally so I can run local tests

**Comment by ijemmao** [comment]
@jibranabsarulislam just wanted to check in if you're still working on this!

**Comment by Abbeyafolaby** [comment]
Hii @ijemmao, can I take up this task if you don't mind?

**Comment by ijemmao** [comment]
@Abbeyafolaby i just assigned you!

**Comment by Abbeyafolaby** [comment]
Alright, I will start working on it. Thanks.

**Comment by jibranabsarulislam** [comment]
Hey @ijemmao,I'm sorry I didn't get back quicker. I wanted to sort out #656 before pushing another PR. I'll keep an eye on the issue and check back on the repo. thank you!

**Comment by ijemmao** [comment]
that's a good point! should I assign #656 to you @jibranabsarulislam - i'm also happy to help look into that bug

**Comment by Abbeyafolaby** [comment]
Alright, Thanks

**Comment by peoray** [comment]
@ijemmao can I pick this up? Seems there's no activity from @Abbeyafolaby

**Comment by ijemmao** [comment]
@peoray I just assigned you!
